doc/HOWTO_RELEASE.rst.txt

```diff
@@ -37,7 +37,7 @@ Supported platforms and versions
 ================================
 
 Python 2.7 and >=3.4 are the currently supported versions when building from
-source.  We test numpy against all these versions every time we merge code to
+source.  We test NumPy against all these versions every time we merge code to
```

Comment:
I don't agree with any of the wording changes before this line

Reply:
Thanks a lot for your review! :)
I'll revert the changes in this file before this line. Shall I also revert the changes in ```C_STYLE_GUIDE.rst.txt``` file? I observed that minor fixes for spelling mistakes and grammatical errors can be done in that file.

Reply:
Missed that this was multiple files. Yes, please do, happy to merge the rest.